server/lib/src/server/mod.rs

```diff
@@ -1537,7 +1559,15 @@ impl<'a> QueryServerWriteTransaction<'a> {
         let filt = filter_all!(f_eq(Attribute::Uuid, udi));
         self.internal_modify(&filt, &modl)
     }
-
+    pub fn set_domain_ldap_allow_unix_pw_bind(
```

Comment:
I don't think you need a dedicated set function here, none of the other domain update bits have it.